azure_arc_servers_jumpstart/docs/azure_arm_linux.md

```diff
@@ -0,0 +1,86 @@
+#  Onboard an Azure Linux Server VM with Azure Arc
+
+The following README will guide you on how onboard a Azure Linux VM on to Azure Arc. An ARM template is provided for the creation of the Azure Resouces, along with an script that will allow you to onboard the Azure VM onto Azure Arc, this step is requiered as Azure VMs are already part of ARM, therefore, the Azure Arc agent cannot be installed following the regular onboarding method. 
```

Comment:
```suggestion
The following README will guide you on how to onboard an Azure Linux VM on to Azure Arc. An ARM template is provided for the creation of the Azure Resouces, along with a script that will allow you to onboard the Azure VM onto Azure Arc, this step is required as Azure VMs are already part of ARM, therefore, the Azure Arc agent cannot be installed following the regular onboarding method. 
```

The onboarding to Arc should be automatic by default. We should also provide a "semi-automated" approach so the user can demo the onboarding itself by manually running the script

Reply:
I did the onboarding by running a script, as automatic as possible. Were you thinking on a different way? Using custom scripts as far as the testing I did is not possible, happy to talk about this